src/pudl/transform/ferc714.py

```diff
@@ -1142,7 +1191,15 @@ def spot_fix_forecast_years_xbrl(df):
         This function also checks that the values for forecast year are within an
         expected range.
         """
-        df = df.astype({"forecast_year": "Int64"})
+        # [2024-aug kmm]
+        # there was some mixup and C004245 put text in their forecast year fields;
+        # excluding those records for now
+        text_in_year_mask = (
+            (df.respondent_id_ferc714_xbrl == "C004245")
+            & (df.report_year == 2024)
+            & (df.net_demand_forecast_mwh.isna())
+        )
+        df = df.loc[~text_in_year_mask].astype({"forecast_year": "Int64"})
```

Comment:
If they revise this filing in the future and fix the bad data, will we still end up excluding it? Or is there a way to gracefully pull it in if it gets fixed? Or maybe we want to raise an error so we can remove this special case?

Reply:
I updated this to raise an error if the data gets fixed upstream so we don't end up with a zombie spot fix. Thanks for filing a ticket with FERC @krivard how did you do that?

Reply:
i think the assert you added is a good thing here to catch if/when this becomes no longer necessary

Reply:
@cmgosnell Okay then I think I'll go ahead and add it to the merge queue.